From 9773e52a108ee1d8645900619a0858bbaf760fc9 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Wed, 14 Jun 2023 06:42:36 +0530
Subject: [PATCH] Fix build issue in iwl7000 driver

Build error seen in iwl7000 driver.

iwl7000 driver using time member which is defined under
CPTCFG_IWLWIFI_DEBUGFS.

Until the core driver tag fixes the issue, make timestamp struct
available even when CPTCFG_IWLWIFI_DEBUGFS is not defined.

Tracked-On: OAM-111096
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 drivers/net/wireless/iwl7000/iwlwifi/fw/runtime.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/wireless/iwl7000/iwlwifi/fw/runtime.h b/drivers/net/wireless/iwl7000/iwlwifi/fw/runtime.h
index b8f8cdfa8fb6..b2fb8295346e 100644
--- a/drivers/net/wireless/iwl7000/iwlwifi/fw/runtime.h
+++ b/drivers/net/wireless/iwl7000/iwlwifi/fw/runtime.h
@@ -145,14 +145,14 @@ struct iwl_fw_runtime {
 			u32 umac_minor;
 		} fw_ver;
 	} dump;
-#ifdef CPTCFG_IWLWIFI_DEBUGFS
+//#ifdef CPTCFG_IWLWIFI_DEBUGFS
 	struct {
 		struct delayed_work wk;
 		u32 delay;
 		u64 seq;
 	} timestamp;
 	bool tpc_enabled;
-#endif /* CPTCFG_IWLWIFI_DEBUGFS */
+//#endif /* CPTCFG_IWLWIFI_DEBUGFS */
 #ifdef CONFIG_ACPI
 	struct iwl_sar_profile sar_profiles[ACPI_SAR_PROFILE_NUM];
 	u8 sar_chain_a_profile;
-- 
2.41.0

